### ISSUE TITLE
Semspect Class names with special chars are not shown correctly

### ISSUE THREAD

**Issue by seebi** [derivo-company/semspect-issues#1]
## Description

Creating a semspect class with an "<" results in a "&lt;" in the interface

## To Reproduce
1. Create a semspect class with a name which has "<" in it

## Expected behavior

name should be shown with normal chars, not with html entities.

## Attachments

<img width="665" alt="Screenshot 2024-08-12 at 16 37 11" src="https://github.com/user-attachments/assets/ad56a657-5688-49c3-9b46-49bd03265049">

## Environment
Include relevant details about your environment, such as:
- Operating System: Linux, dockerized
- SemSpect Version: "RDF v15.0.1"

**Comment by mmopitz** [comment]
Thanks for reporting. 

I did some further testing. 
I found following affected parts (see attached screenshot):

- Exploration Canvas
  -  Group Labels: for owl:Classes (**1**) as well as for SemSpect Classes (**2**)  
  -  ObjectProperty labels (**4**)
- ObjectDossier
  -  label of data/annotationproperties (**3**)

![image](https://github.com/user-attachments/assets/e048cd85-8710-4df8-afe1-cca4bff7307f)

DataSet:

```turtle
@prefix : <http://www.example.org/> .
@prefix owl: <http://www.w3.org/2002/07/owl#> .
@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
@prefix xml: <http://www.w3.org/XML/1998/namespace> .
@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
@prefix foaf: <http://xmlns.com/foaf/0.1/> .
@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .

:ClassWithLabel a owl:Class;
  rdfs:label ">ClassWithLabel<" .

:ClassWithoutLabel a owl:Class .

:object-property-with-label a owl:ObjectProperty;
  rdfs:label ">object-property-with-label<" .

:object-property-without-label a owl:ObjectProperty.

:datatype-property-with-label a owl:DatatypeProperty;
  rdfs:label ">datatype-property-with-label<" .

:datatype-property-without-label a owl:DatatypeProperty.

:individual-with-label a :ClassWithLabel, :ClassWithoutLabel;
  rdfs:label ">individual-with-label<";
  :object-property-with-label _:1 ;
  :object-property-without-label _:1;
  :datatype-property-with-label ">literal<";
  :datatype-property-without-label ">literal<" .

:individual-without-label a :ClassWithLabel, :ClassWithoutLabel .

_:1 a :ClassWithLabel, :ClassWithoutLabel .
```

**Comment by vincentvialard** [comment]
Fixed in version 17.0.0
https://www.semspect.de/rdf-semspect-distribution-17.0.0.zip

**Comment by seebi** [comment]
Thanks for fixing this - we will upgrade to v17 then.
